src/compas/scene/meshobject.py

```diff
@@ -56,11 +56,9 @@ class MeshObject(SceneObject):
     edgecolor = ColorDictAttribute()
     facecolor = ColorDictAttribute()
 
-    def __init__(self, mesh, **kwargs):
-        super(MeshObject, self).__init__(item=mesh, **kwargs)
-        self._mesh = None
+    def __init__(self, **kwargs):
```

Comment:
include `vertexcolor`, `edgecolor`, `facecolor`, ... explicitly instead of processing `**kwargs`